Pattern21.java

```diff
@@ -0,0 +1,42 @@
+package javapatternsmorning;
+
+/**
+ *
+ * @author ErSKS

```

Comment:
Generation of this pattern is a little bit complex. Understand carefully the given code !